clang/Util.d

```diff
@@ -80,6 +80,11 @@ mixin template CX ()
     }
 }
 
+string clangVersion()
+{
+    return clang_getClangVersion().toD;
```

Comment:
I've just been trying this and it doesn't work (it behaves like you said). I'll provide more explanation later.